tests/test_input_matsim_reader.py

```diff
@@ -259,9 +265,15 @@ def test_reading_matsim_output_network():
                                                   'text': '26997928'},
                                    'osm:way:name': {'name': 'osm:way:name', 'class': 'java.lang.String',
                                                     'text': 'Brunswick Place'}}}
+    correct_attributes = {
```

Comment:
Is this expectation based on implicit knowledge of the contents of `matsim_output_network`?

Reply:
yup, I moved this into a fixture to make this more clear